src/components/tree-detail/tree-water-need-unknown.tsx

```diff
@@ -53,39 +53,37 @@ export const TreeWaterNeedUnknown: React.FC<TreeWaterNeedUnknownProps> = ({
 							? i18n.treeDetail.waterNeed.seniorTitle
 							: i18n.treeDetail.waterNeed.unknownTitle}
 					</div>
-					<div className="relative col-start-1 row-start-1 flex flex-row items-center justify-between">
-						<div className="relative">
-							<button
-								onClick={() => {
-									setShowInfoBox(!showInfoBox);
-								}}
-								onMouseMove={() => setShowInfoBox(true)}
-								onMouseOut={() => setShowInfoBox(false)}
-							>
-								<img
-									src="/images/info-icon.svg"
-									alt="Tree Icon"
-									width={24}
-									height={24}
-								/>
-							</button>
-							{showInfoBox && (
-								<div className="text-default absolute right-0 top-8">
-									<Tooltip
-										title={i18n.treeDetail.waterNeed.ageAndWaterHintTitle}
-										content={i18n.treeDetail.waterNeed.ageAndWaterHint}
-									/>
-								</div>
-							)}
-						</div>
-					</div>
 				</div>
 			</div>
 			<div>
 				{treeAgeClassification === TreeAgeClassification.SENIOR
 					? i18n.treeDetail.waterNeed.seniorExplanation
-					: i18n.treeDetail.waterNeed.unknown}
+					: i18n.treeDetail.waterNeed.unknown}{" "}
+				{!isInfoboxVisible && (
+					<button
+						className="text-gdk-blue hover:text-gdk-light-blue font-semibold"
+						onClick={() => setIsInfoboxVisible(true)}
+					>
+						{i18n.treeDetail.waterNeed.readMore}
+					</button>
```

Comment:
unfortunately, I couldn't use our tertiary button here as it would go in a new line due to its height